### ISSUE TITLE
Bug: Unable to connect to large amount of URL's write: operation not permitted

### ISSUE THREAD

**Issue by CEbbinghaus** [qdm12/gluetun#1701]
### Is this urgent?

No

### Host OS

UnraidOS (based on Slackware)

### CPU arch

x86_64

### VPN service provider

NordVPN

### What are you using to run the container

Portainer

### What is the version of Gluetun

Running version latest built on 2023-06-18T11:04:31.032Z (commit 515ae8e)

### What's the problem 🤔

Most connections fail. When manually trying with wget & setting the http proxy it works but for most other connections (from configured apps) it prints the following logs. 

### Share your logs

```plain text
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:36039: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:54925->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:59917: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:58699->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:37309: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:45189->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:34111: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:44570->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:44443: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:52849->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:60471: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:50866->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:51221: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:56813->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:56859: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:60057->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:34541: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:59750->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:57489: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:60487->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:55129: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:60476->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:50541: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:39914->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:33+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:40957: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:59524->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:34+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:59677: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:51582->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:34+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:39031: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:55082->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:34+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:47157: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:38061->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:33335: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:55108->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:39145: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:41679->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:52319: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:33954->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:56359: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:41679->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:39613: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:33954->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:54185: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:55108->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:57347: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:41679->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:53661: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:55108->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:43881: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:43776->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:51299: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:55108->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:50989: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:41679->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:50001: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:35507->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:48705: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:38927->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:33375: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:45514->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:59725: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:55128->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:57287: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:34247->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:40489: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:52141->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:34151: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:56319->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:33193: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:57203->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:55375: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:54720->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:55091: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:32909->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:48419: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:37655->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:33653: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:42218->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:37131: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:37655->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:52233: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:33139->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:57849: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:58678->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:48475: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:58678->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:57627: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:52673->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:54369: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:36250->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:59953: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:42821->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:34443: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:39581->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:34693: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:60977->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:54549: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:40144->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:44675: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:50933->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:37657: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:51866->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:52185: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:60977->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:59601: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:47922->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:33579: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:60977->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:51103: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:35259->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:50115: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:44361->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:52481: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:53996->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:42907: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:40921->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:34457: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:60408->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:59917: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:56485->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:44127: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:46252->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:35615: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:46252->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:54945: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:39823->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:54787: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:56485->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:55665: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:59689->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:47329: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:59689->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:58823: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:34726->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:45445: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:59051->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:51317: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:55316->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:39259: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:53789->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:38789: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:34726->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:47389: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:55102->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:44011: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:46238->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:33645: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:50889->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:52501: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:58116->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:53697: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:38933->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:60423: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:53369->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:50001: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:58060->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:51257: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:49596->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:33783: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:45115->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:54103: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:58557->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:48303: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:41847->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:55653: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:52265->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:52787: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:52265->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:56621: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:53970->1.1.1.1:53: write: operation not permitted
2023-06-23T01:01:35+10:00 WARN [http proxy] cannot process request for client 172.24.0.5:36137: Get "[URL]": dial tcp: lookup [Domain] on 1.1.1.1:53: write udp 172.24.0.3:47180->1.1.1.1:53: write: operation not permitted
2023-06-23T01:02:40+10:00 ERROR [vpn] finding a VPN server: filtering servers: no server found: for VPN wireguard; protocol udp
2023-06-23T01:02:40+10:00 INFO [vpn] retrying in 32m0s
2023-06-23T01:02:52+10:00 INFO [healthcheck] program has been unhealthy for 2m16s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-06-23T01:05:13+10:00 INFO [healthcheck] program has been unhealthy for 2m21s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-06-23T01:07:39+10:00 INFO [healthcheck] program has been unhealthy for 2m26s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-06-23T01:10:10+10:00 INFO [healthcheck] program has been unhealthy for 2m31s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
```

### Share your configuration

```yml
gluetun:
    container_name: gluetun
    image: ghcr.io/qdm12/gluetun:latest
    restart: unless-stopped
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    volumes:
      - /mnt/user/appdata/gluetun:/gluetun
    environment:
      # See https://github.com/qdm12/gluetun/wiki
      - VPN_SERVICE_PROVIDER=nordvpn
      - VPN_TYPE=wireguard
      - WIREGUARD_PRIVATE_KEY=[Private Key]
      - "TZ=${TIMEZONE}"
      # See https://github.com/qdm12/gluetun/wiki/Updating-Servers#periodic-update
      - UPDATER_PERIOD=24h
      - UPDATER_VPN_SERVICE_PROVIDERS=nordvpn

      - SHADOWSOCKS=on
      - HTTPPROXY=on
      - HTTPPROXY_LOG=on 
      - HTTPPROXY_STEALTH=on
```

**Comment by CEbbinghaus** [comment]
Found this but it doesn't seem particularly helpful:
https://community.cloudflare.com/t/1-1-1-1-port-53-intercepted-by-isp/69515/12

**Comment by qdm12** [comment]
> 2023-06-23T01:02:52+10:00 INFO [healthcheck] program has been unhealthy for 2m16s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)

Your VPN is not working, hence anything is blocked by the firewall.
So DNS traffic through the http proxy is blocked `write: operation not permitted`

Check why the VPN doesn't work?

**Comment by qdm12** [comment]
Closing this since this is likely due to the vpn not working only (due to a config problem/auth error), feel free to open another issue.
